src/RA_Dlg_MemBookmark.cpp

```diff
@@ -825,42 +852,49 @@ std::string Dlg_MemBookmark::ImportDialog()
 {
     std::string str;
 
+	// With how repetive this is, it should be it's own function
```

Comment:
more commentary better suited to the PR

Reply:
OK, I'll save that stuff in another file (not PR) so I don't forget.